observer/api.go

```diff
@@ -0,0 +1,119 @@
+package observer
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+// API is object which exposes observer API.
+type API struct {
+	client *http.Client
+	url    string
+}
+
+const (
+	hermesesEndpoint = "api/v1/observed/hermes"
+)
+
+// NewAPI returns a new API instance.
+func NewAPI(url string) *API {
+	return &API{
+		client: &http.Client{
+			Timeout: time.Second * 10,
```

Comment:
if its generic I suggest we use optional configurable timeout

Reply:
Oh, forgot about this. good point. 👍🏻